control_pkg/tests/intregration/integration.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# license removed for brevity
+import rospy
+import rostest
+import unittest
+
+from std_msgs.msg import Float64
+from sensor_msgs.msg import JointState
+from control_msgs.msg import JointControllerState
+import math
+import random
+import numpy as np
+
+
+class intTest(unittest.TestCase):
+	
+	def move_joint(self, pub, jvq):
+		for i in range (7):
+			pub.publish(jvq[i])
```

Comment:
as i understand you publish all values in the one publisher.

Reply:
Solved. I'm using 6 separate publishers in the move_joint function now.